__docs__/wonder-blocks-form/form-utilities.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * Checks if a value is a valid email and returns an error message if it is invalid.
+ * @param value the email to validate
+ * @returns An error message if there is a validation error
+ */
+export const validateEmail = (value: string) => {
+    const emailRegex = /^[^@\s]+@[^@\s.]+\.[^@.\s]+$/;
+    if (!emailRegex.test(value)) {
+        return "Please enter a valid email";
+    }
+};
+
+/**
+ * Checks if a value is a valid phone number and returns an error message if it is invalid.
+ * @param value the phone number to validate
+ * @returns An error message if there is a validation error
+ */
+
```

Comment:
**nit(really super nit):** There's an extra line here. I wouldn't bother about this, but I'm not sure if that extra line messes up with the info that we would see in the IDE (when trying to call this fn).

Reply:
Updated! VS code was still showing the docs with the new line, but cleaned it up anyways!